layers/modules/fast_mask_iou.py

```diff
@@ -0,0 +1,33 @@
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+#locals
+from data.config import Config
+from utils.functions import make_net
+
+
+#  As of March 10, 2019, Pytorch DataParallel still doesn't support JIT Script Modules
+use_jit = torch.cuda.device_count() <= 1
+if not use_jit:
+    print('Multiple GPUs detected! Turning off JIT.')
+
+ScriptModuleWrapper = torch.jit.ScriptModule if use_jit else nn.Module
+script_method_wrapper = torch.jit.script_method if use_jit else lambda fn, _rcn=None: fn
```

Comment:
This part should probably go in its own file, since it should only be loaded once and might need to be used in multiple places.

Reply:
not sure why that's important, but like this fccd89b ?